app/controllers/application_controller.rb

```diff
@@ -25,6 +25,18 @@ class ApplicationController < ActionController::Base
   # includes a security token
   protect_from_forgery with: :exception
 
+  def logout
```

Comment:
The cop for this is disabled. 
`# This cop does not consider action inheritance, therefore disabled
Rails/LexicallyScopedActionFilter:
  Enabled: false`